e2e/sg-creation.e2e-spec.ts

```diff
@@ -0,0 +1,115 @@
+    sglist.clickFirewallMenu();
+    sglist.clickCreateSG();
+    sglist.waitDialogModal();
+    page = new SGCreation();
+    page.setSGDescription(page.description);
+    page.clickADDRules();
+    page.verifyBuildNewSGModal();
+    page.selectTemplate(0);
+    page.getNetworkRules().then(expectedRules => {
+      expect(page.getSaveRulesButton().isEnabled()).toBeTruthy('Button Save should be enabled');
+      page.getSaveRulesButton().click();
+      expect(page.buttonCreateSGEnabled()).toBeFalsy('Button Create should be disabled');
+      page.setSGName(page.name);
+      expect(page.buttonCreateSGEnabled()).toBeTruthy('Button Create should be enabled');
+      page.clickCreateSGButton();
+      page.waitCreatingTemplate(page.name);
+      sglist.clickSGActionBox();
+      sglist.clickSGRules();
+      sglist.getElementFromRules().then(actualRules => {
+        expect(actualRules).toEqual(expectedRules);
+      });
+      sglist.closeSGRules();
+    });
+  });
+
+  it('Verify custom SG card: name, description', () => {
+    expect(sglist.getSGNameCard()).toEqual(page.name);
+    expect(sglist.getSGDescriptionCard()).toEqual(page.description);
+  });
+
+  it('Verify custom Sidebar: name, description , type', () => {
+    sglist.clickOpenSidebar(page.name);
+    expect(sgsidebar.getSGName()).toEqual(page.name);
+    expect(sgsidebar.getSGDescription()).toEqual(page.description);
+    expect(sgsidebar.getSGType()).toEqual('custom-template');
+    sgsidebar.clickTagTab();
+    sgsidebar.clickChekbox();
+    expect(sgsidebar.getTagKey('csui.security-group.type').isPresent()).toBeTruthy(
+      'csui.security-group.type',
+    );
+    expect(sgsidebar.getTagValue('custom-template').isPresent()).toBeTruthy('custom-template');
+    sgsidebar.clickChekbox();
+    sgsidebar.clickClose();
+  });
+
+  it('Create template in Shared Security Group, verify rules', () => {
+    sglist.clickSharedTab();
+    sglist.clickCreateSG();
+    sglist.waitDialogModal();
+    page = new SGCreation();
+    page.setSGDescription(page.description);
+    page.clickADDRules(); // click ADD
+    page.selectTemplate(0);
+    page.getNetworkRules().then(expectedRules => {
+      expect(page.getSaveRulesButton().isEnabled()).toBeTruthy('Button Save should be enabled');
+      page.getSaveRulesButton().click();
+      expect(page.buttonCreateSGEnabled()).toBeFalsy('Button Create should be disabled');
+      page.setSGName(page.name);
+      expect(page.buttonCreateSGEnabled()).toBeTruthy('Button Create should be enabled');
+      page.clickCreateSGButton();
+      page.waitCreatingTemplate(page.name);
+      sglist.clickSGActionBox();
+      sglist.clickSGRules();
+      sglist.getElementFromRules().then(actualRules => {
+        expect(actualRules).toEqual(expectedRules);
+      });
+      sglist.closeSGRules();
+    });
+  });
+
+  it('Verify shared SG card: name, description', () => {
+    expect(sglist.getSGNameCard()).toEqual(page.name);
```

Comment:
The error got because of Default group exists. Need to fix the order of the created shared group. 
 -  Expected 'default' to equal 'e2ee96l6wjha'.
  - Expected 'Default Security Group' to equal 'e2eodpkfi11d'.